webrender/src/render_backend.rs

```diff
@@ -401,8 +401,7 @@ impl RenderBackend {
                         notifier.as_mut()
                                 .unwrap()
                                 .as_mut()
-                                .unwrap()
-                                .pipeline_size_changed(pipeline_id, Some(new_size));
+                                .unwrap();
```

Comment:
This code doesn't do anything now, since it isn't used to notify anything. We can remove all of the code that compares the sizes of pipelines and stores them for later.